src/Akeneo/Connectivity/Connection/front/src/webhook/components/EventLogList.tsx

```diff
@@ -84,7 +86,10 @@ export const EventLogList: FC<{connectionCode: string}> = ({connectionCode}) =>
                 </Table.Header>
                 <Table.Body ref={scrollContainer}>
                     {logs.map(({timestamp, level, message, context}, index) => (
-                        <Table.Row key={index} onClick={() => undefined}>
+                        <ExpandableTableRow
+                            key={index}
+                            contentToExpand={<FormattedJSON>{context}</FormattedJSON>}
```

Comment:
I think that we should make the expandable row **lazy load** its content to not pre-render the formatted json for each line as this could have a big impact on performance.
```suggestion
                            contentToExpand={() => <FormattedJSON>{context}</FormattedJSON>}
```

Something like the ant design expandable row:
https://ant.design/components/table/#components-table-demo-expand

Reply:
After reflexion I don't think it's necessary here as the component is just transpiled to a function and will not be called if the row is not expanded